package.json

```diff
@@ -83,6 +84,7 @@
     "@types/hammerjs": "^2.0.38",
     "@types/hoist-non-react-statics": "^3.3.1",
     "@types/jest": "^27.0.3",
+    "@types/node": "^20.12.2",
```

Comment:
I remember there being problems where `node` types would be used by GH instead of `react` which was causing TS errors, so it would be best to remove it.

Reply:
Done in [32595fb](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/32595fb8a63e99724233921ba9ceb0ccfaf4eeae)